demos/src/demo.scss

```diff
@@ -0,0 +1,21 @@
+$o-typography-is-silent: false;
+@import 'o-typography/main';
+
+$o-topper-is-silent: false;
+@import '../../main';
+
+@include oTopperThemes;
+
+body {
+	@include oColorsFor(page, background);
+}
+
+.o-topper {
+	// TODO: add to next-article, discuss whether o-topper should do it itself
+	-webkit-font-smoothing: antialiased; // sass-lint:disable-line no-vendor-prefixes
```

Comment:
Is this something `o-normalise` could handle as a dependancy of the demo?
https://github.com/Financial-Times/o-topper/pull/1/files#diff-4f8e4daa8d64113f85a8b1ce831b2685R21